result/isislsp.yaml

```diff
@@ -0,0 +1,380 @@
+            LSP ID consists of the System ID of a neighbor, the Pseudonode ID, and the LSP number.
+            The last two bytes represent Pseudonode ID and LSP number respectively.
+            A pseudonode is a logical representation of the LAN which is generated by a Designated Intermediate System (DIS)
+            on a LAN segment. If one LSP exceeds the maximum LSP size then it is sent in another LSP with the LSP number incremented by one.
+            A router's learned LSP gets refreshed by 'remaining_lifetime'. Then the sequence number is incremented by 1.
+          type: string
+        pdu_type:
+          description: >-
+            Link State PDU type.
+          type: string
+          enum: [level_1, level_2]
+        remaining_lifetime:
+          description: >-
+            Remaining lifetime in seconds before the LSP expiration.
+          type: integer
+          format: int64
+          minimum: 0
+          maximum: 4294967295
+        sequence_number:
+          description: >-
+            Sequence number of the LSP.
+          type: integer
+          format: int64
+          minimum: 0
+          maximum: 4294967295
+        pdu_length:
+          description: >-
+            Total length of the LSP.
+          type: integer
+          minimum: 0
+          maximum: 65535
+        flags:
+          description: >-
+            LSP Type-Block flags.
+          $ref: '#/components/schemas/IsisLsp.Flags'
+        is_type:
+          description: |-
+            IS Type - bits 1 and 2 indicate the type of Intermediate System.
+            1 - ( i.e. bit 1 set) Level 1 Intermediate system.
+            2 - Unused value.
+            3 - (i.e. bits 1 and 2 set) Level 2 Intermediate system. 
+          type: integer
+          minimum: 1
+          maximum: 3
+        tlvs:
+          description: |-
+            This container defines Link State PDU State TLVs.
+          $ref: '#/components/schemas/IsisLsp.Tlvs'
+
+    IsisLsp.Tlvs:
+      description: >-
+        This contains all TLVs. As per ISIS-Yang model Hostname, IS-Reachability, Extended-IS-Reachability etc. are list.
+      type: object
+      properties:
+        hostnames:
+          description: >-
+            This container defines TLV 137 for the hostname.
+          type: array
+          items:
+            type: string
+        is_reachabilities:
+          description: |-
+            Array of list of ISIS neighbors and attributes.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.IsReachability'
+        extended_is_reachabilities:
+          description: |-
+            Array of list of ISIS neighbors and attributes.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.ExtendedIsReachability'
+        ipv4_internal_reachabilities:
+          description: |-
+            Array of list of ISIS neighbors and attributes. list of IPv4 internal reachability information.
```

Comment:
I thought at this level we should be having array of TLVs of type ipv4_internal_reachability ?

Reply:
Array of list of IPv4 Internal Reachability TLVs present in this LSP.